src/utilcode/util.cpp

```diff
@@ -1288,7 +1288,15 @@ int GetCurrentProcessCpuCount()
     // the case where there are more than 64 processors, so we will return a
```

Comment:
@helloguo please see the comment here: 
The GC also cannot (currently) handle the case where there are more than 64 processors, so we will return amaximum of 64 here. So the limitation is intentional and is the same for Windows and Unix.